akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -356,30 +398,30 @@ object DirectoryListing {
   def directoryMarshaller(renderVanityFooter: Boolean): ToEntityMarshaller[DirectoryListing] =
     Marshaller.StringMarshaller.wrapWithEC(MediaTypes.`text/html`) { implicit ec ⇒ listing ⇒
       val DirectoryListing(path, isRoot, files) = listing
```

Comment:
Can't change `DirectoryListing#files: Seq[File]` to `Seq[Path]` because it's public.